mmv1/third_party/terraform/transport/config.go.tmpl

```diff
@@ -1156,6 +1224,26 @@ type StaticTokenSource struct {
       oauth2.TokenSource
 }
 
+// getExternalAccountConfig returns an externalaccount.Config based on the Config object.
+// The external account config is intended to be used to create a token source.
+func (c *Config) getExternalAccountConfig(clientScopes []string) externalaccount.Config {
+	eaConfig := externalaccount.Config{
+		Audience:         c.ExternalCredentialsHcpTerraform.Audience,
+		SubjectTokenType: "urn:ietf:params:oauth:token-type:jwt",
```

Comment:
This is an example of something that we hardcode that's compatible with HCP Terraform